tests/notofont/LICENSE_OFL.txt

```diff
@@ -0,0 +1,92 @@
+This Font Software is licensed under the SIL Open Font License,
```

Comment:
I think the new test should go under `tests/third_party/notofont` - is that right @sbc100 ?

Reply:
Moved it to `third_party` folder.